index.html

```diff
@@ -81,11 +81,11 @@
   
           <div class="slides-description-list">
             <div class="slide-description">Universe, 2016</div>
-            <!-- <div class="slide-description">Nikola Tesla Tower, 2020</div>
```

Comment:
This does not belong here, but we can leave it as is for now.

Reply:
Ups...